build.xml

```diff
@@ -1,74 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<project basedir="." default="build" name="mmj2">
-    <property environment="env"/>
-    <property name="ECLIPSE_HOME" value="C:/Program Files/eclipse4"/>
-    <property name="debuglevel" value="source,lines,vars"/>
-    <property name="target" value="1.8"/>
-    <property name="source" value="1.8"/>
-    <path id="mmj2.classpath">
-        <pathelement location="classes"/>
-    </path>
-    <target name="init">
-        <mkdir dir="classes"/>
-        <mkdir dir="doc/javadoc"/>
-        <copy includeemptydirs="false" todir="classes">
-            <fileset dir="src">
-                <exclude name="**/*.launch"/>
-                <exclude name="**/*.java"/>
-            </fileset>
-        </copy>
-    </target>
-    <target name="clean">
-        <delete dir="classes"/>
-        <delete dir="doc/javadoc"/>
-    </target>
-    <target depends="clean" name="cleanall"/>
-    <target depends="init" name="build-project">
```

Comment:
Uh, are you sure you want to delete the ant build script? I don't think you have replicated all the targets here. Ideally the ant build would continue to work, with only some path changes.

Reply:
Is it possible to revert [just that commit](https://github.com/digama0/mmj2/pull/65/commits/e1ab7fbcbd2eac376e9bda8fdfe28f72a95a1f14)?

Reply:
I'll remove the commit and try to make the ant script work with the new path.